### DISCUSSION TITLE
When to use <SelectList>

### DISCUSSION THREAD

**Discussion by crisshaker** [ariakit/ariakit#2521] [Questions] [answered]
Is `<SelectList>` intended for UIs like this? A select that is always visible and doesn't have any sort of trigger?

<img width="757" alt="image" src="https://github.com/ariakit/ariakit/assets/36159205/9a19e740-6720-4178-8779-226098637727">

**Comment by diegohaz** [comment, marked as answer]
We don't have tests for that. But, yes, that would be a valid use case.

**Reply by crisshaker** [reply]
alright... thanks, man!
